lib/config.js

```diff
@@ -16,6 +18,13 @@ cfg.routes = {
   basePath
 };
 
-config.validation.schema.paths.push(
-  path.join(__dirname, '..', 'schemas')
-);
+cfg.kms = {
+  private: {
+    host: config.server.domain,
+    port: config.server.port,
+  },
+  public: {
+    host: config.server.domain,
+    port: config.server.port,
```

Comment:
These need to be computed vars.

Reply:
If these are made into computed vars, then people who are changing `config.server.domain`, etc. don't have to make sure to also reset these variables as well. When some config var depends on another, the best practice is to make it a computed var.